source/dubregistry/registry.d

```diff
@@ -408,8 +411,31 @@ class DubRegistry {
 		dbver.info = info.info;
 
 		try {
-			rep.readFile(reference.sha, InetPath("/README.md"), (scope input) { dbver.readme = input.readAllUTF8(); });
-		} catch (Exception e) { logDiagnostic("No README.md found for %s %s", dbpack.name, ver); }
+			auto files = rep.listFiles(reference.sha, InetPath("/"));
+			// check exactly for readme.me
+			ptrdiff_t readme;
+			readme = files.countUntil!(a => a.type == RepositoryFile.Type.file && a.path.head.name.asUpperCase.equal("README.MD"));
+			if (readme == -1) {
+				// check exactly for readme
+				readme = files.countUntil!(a => a.type == RepositoryFile.Type.file && a.path.head.name.asUpperCase.equal("README"));
+			}
+			if (readme == -1) {
+				// check for all other readmes such as README.txt, README.jp.md, etc.
+				readme = files.countUntil!(a => a.type == RepositoryFile.Type.file && a.path.head.name.asUpperCase.startsWith("README"));
```

Comment:
Nice, somewhat problematic that we cannot render them though.

Reply:
well at least support for README, Readme, readme, ReadMe, etc now. i18n is something we might eventually look into, I have seen multiple japanese repositories with different readme names with this README.en.md pattern